connector/src/main/java/org/geysermc/connector/entity/type/EntityType.java

```diff
@@ -174,17 +180,33 @@
      */
     ENDER_DRAGON_PART(EnderDragonPartEntity.class, 32, 0, 0, 0, 0, "minecraft:armor_stand");
 
+    /**
+     * A list of all Java identifiers for use with command suggestions
+     */
+    public static final String[] ALL_JAVA_IDENTIFIERS;
     private static final EntityType[] VALUES = values();
 
-    private Class<? extends Entity> entityClass;
+    static {
+        List<String> allJavaIdentifiers = new ArrayList<>();
+        for (EntityType type : values()) {
+            if (type == AGENT || type == BALLOON || type == CHALKBOARD || type == NPC || type == TRIPOD_CAMERA || type == ENDER_DRAGON_PART) {
```

Comment:
We should probably just remove most of these since they are not part of normal Bedrock edition.

Reply:
```suggestion
            if (type == ENDER_DRAGON_PART) {
```

Reply:
I didn't remove these because they might be helpful for future API usage - spawn the entities only for Bedrock.